.github/workflows/build.yaml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "2.5"
+  pull_request:
+  workflow_dispatch:
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.workflow }}-${{ github.event.number || github.head_ref || github.run_id || github.sha }}
+  cancel-in-progress: true
+env:
+  JAVA_VERSION: 11
```

do you mind to step back to version `8`? we changed our mind

```suggestion
  JAVA_VERSION: 8
```